docs/imported/community/guide.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Kubernetes Contributor Guide
+owner: sig-contributor-experience
+---
+
+## Disclaimer
+Hello! This is the starting point for our brand new contributor guide, currently underway as per [issue#6102](https://github.com/kubernetes/website/issues/6102) and in need of help. Please be patient, or fix a section below that needs improvement, and submit a pull request!
+
+Many of the links below should lead to relevant documents scattered across the community repository. Often, the linked instructions need to be updated or cleaned up.
+
+* If you do so, please move the relevant file from its previous location to the community/contributors/guide folder, and delete its previous location.
+* Our goal is that all contributor guide specific files live in this folder.
+
+Please find _Improvements needed_ sections below and help us out.
+
+For example:
+
+_Improvements needed_
+* kubernetes/community/CONTRIBUTING.md -> Needs a rewrite
+
+* kubernetes/community/README.md -> Needs a rewrite
+
+* Individual SIG contributing documents -> add a link to this guide
+
+# Welcome
+
+Welcome to Kubernetes! This document is the single source of truth for how to contribute to the code base. Please leave comments / suggestions if you find something is missing or incorrect.
+
+-   [Before you get started](#before-you-get-started)
+    -   [Sign the CLA](#sign-the-cla)
+    -   [Setting up your development
+        environment](#setting-up-your-development-environment)
+    -   [Community Expectations](#community-expectations)
+        -   [Code of Conduct](#code-of-conduct)
+        -   [Thanks](#thanks)
+-   [Your First Contribution](#your-first-contribution)
+    -   [Find something to work on](#find-something-to-work-on)
+        -   [Find a good first topic](#find-a-good-first-topic)
+        -   [Learn about SIGs](#learn-about-sigs)
+        -   [File an Issue](#file-an-issue)
+-   [Contributing](#contributing)
+    -   [Communication](#communication)
+    -   [GitHub workflow](#github-workflow)
+    -   [Open a Pull Request](#open-a-pull-request)
+    -   [Code Review](#code-review)
+    -   [Testing](#testing)
+    -   [Security](#security)
+    -   [Documentation](#documentation)
+    -   [Issues Management or Triage](#issues-management-or-triage)
+-   [Community](#community)
+    -   [Events](#events)
+        -   [Meetups](#meetups)
+        -   [KubeCon](#kubecon)
+    -   [Mentorship](#mentorship)
+
+# Before you get started
+
+## Sign the CLA
+
+Before you can contribute, you will need to sign the [Contributor License Agreement](/CLA.md).
+
```

Comment:
After the content is imported to kubernetes.io, there are a lot of broken links. It's because of relative link targets like (/CLA.md).

Reply:
Should we be encouraging full URL links in the community repo?

Reply:
@abiogenesis-now is going to add in code to rewrite the URLs.

Reply:
PR is here https://github.com/kubernetes/website/pull/6926